spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/direct/BigQueryRDDContext.java

```diff
@@ -84,7 +84,7 @@ public scala.collection.Iterator<InternalRow> compute(Partition split, TaskConte
     SparkEnv.get().metricsSystem().registerSource(sparkMetricsSource);
     BigQueryStorageReadRowsTracer tracer =
         bigQueryTracerFactory.newReadRowsTracer(
-            Joiner.on(",").join(streamNames), sparkMetricsSource);
+            Joiner.on(",").join(streamNames), sparkMetricsSource, Optional.empty());
```

Comment:
Add a comment that those metrics are not supported in DSv1 as well as the reasons